lib/Ed25519VerificationKey2020.js

```diff
@@ -238,9 +268,10 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     const buffersEqual = _isEqualBuffer(this._publicKeyBuffer,
       fingerprintBuffer.slice(2));
```

Comment:
```suggestion
    // non-constant time `_isEqualBuffer` is safe to use here because only
    // public values are being compared
    const buffersEqual = _isEqualBuffer(
      this._publicKeyBuffer, fingerprintBuffer.slice(2));
```